app/models/sandbox.rb

```diff
@@ -0,0 +1,5 @@
+class Sandbox < ApplicationRecord
+  belongs_to :jurisdiction
```

Comment:
add the `has_many` for `permit_applications` and `jurisdiction_template_version_customizations`